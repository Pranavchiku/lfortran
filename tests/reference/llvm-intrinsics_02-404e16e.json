{
    "basename": "llvm-intrinsics_02-404e16e",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_02.f90",
    "infile_hash": "62898f6506eceb32aa540718b4d9de7cd0f9942d9cf5eb181748eba9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-intrinsics_02-404e16e.stdout",
    "stdout_hash": "d6eaa901e76e82017ec18ad6a58e6a46cd6bec94850b20331cff0aee",
    "stderr": "llvm-intrinsics_02-404e16e.stderr",
    "stderr_hash": "e7c863814d94513b40e7aa5395c1efa974ae674a45000be2004261a4",
    "returncode": 0
}
