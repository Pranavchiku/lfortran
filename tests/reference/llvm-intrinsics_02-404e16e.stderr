warning: Implied save feature
 --> tests/../integration_tests/intrinsics_02.f90:2:1
  |
2 | real :: x
  | ^^^^^^^^^ Assuming save attribute implicitly for variables
